From 8a7634c1bf4222ecff854e2da421dd0d434843ae Mon Sep 17 00:00:00 2001
From: jinghua <jinghua@marvell.com>
Date: Tue, 28 Jun 2016 21:59:27 +0800
Subject: [PATCH 1234/1240] a3700: clean up armada3700.h

- remove word 'palladium' from comment about enable SPI driver,
  since it is not only for Palladium, but also for Asic as well.
- remove CONFIG_SYS_PCI_64BIT define since there has already been
  one in mvebu-common.h

Change-Id: I214b062f5fbd7d1c12657eadd9f66a4768365174
Signed-off-by: jinghua <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30763
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 include/configs/armada3700.h | 29 +----------------------------
 1 file changed, 1 insertion(+), 28 deletions(-)

diff --git a/include/configs/armada3700.h b/include/configs/armada3700.h
index 9a7d43c..ee47772 100644
--- a/include/configs/armada3700.h
+++ b/include/configs/armada3700.h
@@ -27,24 +27,6 @@
 #undef CONFIG_GICV2
 #define CONFIG_GICV3
 
-/*
-#define MV_INCLUDE_SDIO
-#define MV_INCLUDE_INTEG_SATA
-#define MV_INCLUDE_CLK_PWR_CNTRL
-
-#define MV_INCLUDE_RCVR
-#define MV_INCLUDE_NAND
-#define MV_INCLUDE_SPI
-#define MV_INCLUDE_XOR
-#define MV_INCLUDE_SATA
-#define MV_INCLUDE_USB
-#define CONFIG_MV_ETH_NETA
-#define MV_INCLUDE_GPIO
-#define MV_INCLUDE_I2C
-#define MV_INCLUDE_RTC
-#define MV_INCLUDE_PEX
-*/
-
 /* Plaform */
 #define CONFIG_MARVELL
 
@@ -55,11 +37,7 @@
  */
 #define COUNTER_FREQUENCY	(12500000)
 
-/* DRAM detection stuff */
-#define MV_DRAM_AUTO_SIZE
-#define CONFIG_SYS_DRAM_BANKS		1
-
-/* SPI flash for Palladium */
+/* SPI driver support */
 #ifdef CONFIG_MVEBU_SPI
 
 #define CONFIG_MVEBU_SPI_A3700
@@ -81,9 +59,4 @@
  */
 #include "mvebu-common.h"
 
-/* need board_early_init_f to invoke init_mbus, before dram_init */
-#define CONFIG_BOARD_EARLY_INIT_F
-
-#define CONFIG_SYS_PCI_64BIT	/* enable 64-bit PCI resources */
-
 #endif /* __ARMADA_3700_H */
-- 
1.9.1

